### DISCUSSION TITLE
什么值得买readme里面配置圈X写错了

### DISCUSSION THREAD

**Discussion by fmz200** [blackmatrix7/ios_rule_script#341] [Show and tell]
复制到圈X里面更新脚本提示404，检查看到了里面没有checkin这个文件，应该是singin这个文件，望更正。
没有这个：https://raw.githubusercontent.com/blackmatrix7/ios_rule_script/master/script/smzdm/smzdm_checkin.qxrewrite；
应该是这个：https://raw.githubusercontent.com/blackmatrix7/ios_rule_script/master/script/smzdm/smzdm_signin.qxrewrite。

**Comment by blackmatrix7** [comment]
谢谢反馈，README是旧的，之前忘了修改，现在已经改掉了。
